src/LogBuffer.cpp

```
@@ -0,0 +1,68 @@
+#include "LogBuffer.h"
+#include <stdarg.h>
+#include <cstring>
+
+LogBuffer::LogBuffer() {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: buffer_, mutex_

(cppcoreguidelines-pro-type-member-init)